cfgov/cal/views.py

```diff
@@ -11,19 +11,18 @@
 from django.http import HttpResponse, HttpResponseRedirect
 from django.shortcuts import render, render_to_response
 from django.template.loader import get_template
+from django.conf import settings
+
 from httplib import BadStatusLine
 
 from cal.forms import CalendarFilterForm, CalendarPDFForm
 from cal.models import CFPBCalendarEvent
 
 
-## TODO: Update to python 3 when PDFreactor's python wrapper supports it.
-if six.PY2:
-    try:
-        sys.path.append(os.environ.get('PDFREACTOR_LIB'))
-        from PDFreactor import *
-    except ImportError:
-       PDFreactor = None
+try:
+    from PDFreactor import *
+except ImportError:
+    PDFreactor = None
```

Comment:
basically, the PDFReactor python lib isn't distributed via pip and friends, it's bundled with PDFReactor itself. This is a way to make cfgov-refresh work without having PDFReactor installed.